lib/segment/src/common/utils.rs

```diff
@@ -220,6 +220,95 @@ pub fn get_value_from_json_map_opt<'a>(
     }
 }
 
+pub fn set_value_to_json_map<'a>(
```

Comment:
this function requires explicit testing

Reply:
Having openapi integration tests is nice, but it would be better to implement unit-tests here as well